### PR TITLE
Update pipeline and pipelineruns to build image for multiplatform

### PR DESCRIPTION
**Pull request by savitaashture**

### CODE REVIEW

**Comment by savitaashture** [comment]
/cancel tektoncd-triggers-main-eventlistenersink-on-pull-request

File: .tekton/tektoncd-triggers-main-controller-pull-request.yaml [hunk 1]
```diff
@@ -33,27 +33,23 @@ spec:
     value: 5d
   - name: dockerfile
     value: openshift/dockerfiles/controller.Dockerfile
-  - name: build-source-image
-    value: "true"
-  - name: hermetic
-    value: "true"
```
Comment (vdemeester) [inline .tekton/tektoncd-triggers-main-controller-pull-request.yaml:39]: Is it expected to remove this ? 🤔

Comment (savitaashture) [reply .tekton/tektoncd-triggers-main-controller-pull-request.yaml:39]: I did set those as default values

File: .tekton/docker-build.yaml [hunk 2]
```diff
@@ -33,7 +50,7 @@ spec:
     description: Skip checks against built image
     name: skip-checks
     type: string
-  - default: "false"
+  - default: "true"
```
Comment (savitaashture) [inline .tekton/docker-build.yaml:53]: @vdemeester here i made `false` to `true`

File: .tekton/docker-build.yaml [hunk 3]
```diff
@@ -49,10 +66,14 @@ spec:
     description: Image tag expiration time, time values could be something like
       1h, 2d, 3w for hours, days, and weeks, respectively.
     name: image-expires-after
-  - default: "false"
+  - default: "true"
```
Comment (savitaashture) [inline .tekton/docker-build.yaml:69]: similary here as well and removed from pipelinerun yaml

**Comment by savitaashture** [comment]
closing this as automation done through hack repo

**Pull request closed without merging**